src/instructlab/lab.py

```diff
@@ -878,6 +878,16 @@ def convert(self, value, param, ctx) -> "torch.device":
     show_default=True,
     help="model name to use in training",
 )
+@click.option(
+    "--override-training-args",
+    default="{}",
+    show_default=True,
+    hidden=True,
+    help=(
+        "Additional arguments for training (json string)"
+        "e.g. '--override-training-args {\"gradient_accumulation_steps\"=8}'"
```

Comment:
```suggestion
        "e.g. '--override-training-args {\"gradient_accumulation_steps\"=8}'"
         "e.g. '--override-training-args $(< override_train_args.json)'"
```

Depending on the complexity and length I believe it's important to show user that a file can be used too.